src/Umbraco.Cms.ManagementApi/Controllers/ExamineManagement/SearchersExamineManagementController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Examine;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.Infrastructure.Examine;
+using Umbraco.Cms.ManagementApi.ViewModels.ExamineManagement;
+using Umbraco.Cms.ManagementApi.ViewModels.Pagination;
+using Umbraco.Extensions;
+
+namespace Umbraco.Cms.ManagementApi.Controllers.ExamineManagement;
+
+[ApiVersion("1.0")]
+public class SearchersExamineManagementController : ExamineManagementControllerBase
+{
+    private readonly IExamineManager _examineManager;
+
+    public SearchersExamineManagementController(IExamineManager examineManager) => _examineManager = examineManager;
+
+    /// <summary>
+    ///     Get the details for searchers
+    /// </summary>
+    /// <returns></returns>
+    [HttpGet("searchers")]
+    [MapToApiVersion("1.0")]
+    [ProducesResponseType(typeof(PagedViewModel<ExamineIndexModel>), StatusCodes.Status200OK)]
+    // This endpoint for now will throw errors if the ExamineIndexViewModel ever has providerProperties defined
+    // This is because System.Text.Json cannot serialize dictionary<string, object>
+    // This has been fixed in .NET 7, so this will work when we upgrade: https://github.com/dotnet/runtime/issues/67588
+    public async Task<PagedViewModel<ExamineIndexViewModel>> GetSearcherDetails(int skip, int take)
```

Comment:
`GetSearcherDetails` should probably be `Searchers` to follow the modus of naming endpoints and routes the same?